api/read.rb

```diff
@@ -0,0 +1,4 @@
+post "#{APIPREFIX}/users/:user_id/read" do |user_id|
```

Comment:
It is not bad.  I was just wondering.  Thank you for the explanations.  

I think the others that were broken out were against both threads and comments so they required a cross-cutting file to stick together.  For this one, since it is user only, I think I'd move it to users.rb.  Thanks.